include/triton/Conversion/TritonGPUToLLVM/TargetInfoBase.h

```diff
@@ -98,6 +98,10 @@ class TargetInfoBase {
   // llvm.
   virtual void storeOpAnnotation(triton::gpu::LocalStoreOp op,
                                  size_t localStoreOpCount, Type type) const {}
+  // Adds alias information to local loads to help llvm interleave with async
+  // operations
+  virtual void setAsyncAliasScopes(triton::gpu::LocalLoadOp localLoadOp,
```

Comment:
What about naming this as `localLoadOpAnnotation` to be similar like the above? Also can you explain in the comment what the `llLoadOp` is expected to be. Also, redundant `;` at the end.